analyses/tcga-capture-kit-investigation/run-investigation.sh

```diff
@@ -2,62 +2,37 @@
 set -eo pipefail
 
 # set the analyses folder as workdir variable
-WORKDIR='analyses/tcga-capture-kit-investigation'
-
+#WORKDIR='analyses/tcga-capture-kit-investigation'
+WORKDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # retrieve the TCGA's exome capture kit from GDC's file API endpoint
 # this will generate tcga-capture_kit-info.tsv file under the result folder
 python3 $WORKDIR/scripts/get-tcga-capture_kit.py
 
-
 # download all BED from tcga-capture_kit-info.tsv and add chr prefix
 sed 1d $WORKDIR/results/tcga-capture_kit-info.tsv \
 | cut -f3 | tr "\|" "\n" | sort -u \
 | while read i
 do 
     filename=`basename $i`
-    curl -s $i | awk '{print "chr"$0}' > scratch/$filename
+    curl -s $i | awk '{print "chr"$0}' > ../../scratch/$filename
```

Comment:
have you tested this in the container for this? will the `../../scratch/` be compatible with the ci run? https://github.com/tkoganti/OpenPBTA-analysis/blob/Teja_tmbQC/.circleci/config.yml#L209

Reply:
This works according to the .sh script in ependymoma [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-EPN/run-molecular-subtyping-EPN.sh) Let me know if you want me to change these paths back to what they were in your original script